### ISSUE TITLE
Dropdown: When the list box is closed, use the arrow key to pick option, the option text isn’t read aloud.

### ISSUE THREAD

**Issue by gggjtgxing** [OfficeDev/office-ui-fabric-react#675]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Describe the issue:

When the list box is closed, use the arrow key to pick option, the option text isn’t read aloud.

#### Actual behavior:

list box is opened: read aloud selected option text when use the arrow keys to pick
list box is closed: read nothing when use the arrow keys to pick 

#### Expected behavior:

When the list box is closed, use the arrow key to pick option, the option text should be read aloud.

### If applicable, please provide a codepen repro:
This is a good example for dropdown accessibility
http://oaa-accessibility.org/example/10/

**Comment by dzearing** [comment]
@eliblock12 can you check this? You added focus support I believe for screenreader accessibility when the dropdown is expanded, but it seems it's not working when not expanded. I thought this was.
